locales/en/messages.json

```diff
@@ -3594,7 +3594,11 @@
     },
     "osdSetupPreviewSelectProfileTitle": {
         "message": "Preview for",
-        "description": "Label of the selector for the OSD Profile in the preview"
+        "description": "Label of the selector for the OSD Profile in the preview. KEEP IT SHORT!!!"
+    },
+    "osdSetupPreviewForTitle": {
+        "message": "Changing here the profile or the font will NOT change the profile or the font in the flight controller, only affects the preview window. If you want to change it, you must use the '$t(osdSetupSelectedProfileTitle.message)' option or the '$t(osdSetupFontManager.message)' button.",
```

Comment:
Sorry to be nitpicking here, but I think the understadability of this will be greatly improved if we add this at the end: '...or the '$t(osdSetupFontManager.message)' button respectively.'.

Reply:
No problem. Fixed ;)